caffeine/src/main/java/com/percolate/caffeine/ViewUtils.java

```diff
@@ -14,14 +14,17 @@
  * <h2>Utility methods that make working with views easier, less error prone, and more concise.</h2>
  * <p/>
  * <h3>Common uses:</h3>
+ * <code>LinearLayout linearLayout = ViewUtils.{@link #findViewById findViewById}(this, R.id.my_linear_layout);</code> //<b>no more casting!</b><br />
```

Comment:
Move below `findViewById` with `context` examples.  Those are more common and this doc pops up in the IDE.